pkg/cortex/runtime_config.go

```diff
@@ -78,9 +79,14 @@ func (l runtimeConfigLoader) load(r io.Reader) (interface{}, error) {
 		return nil, errMultipleDocuments
 	}
 
-	for _, ul := range overrides.TenantLimits {
-		if err := ul.Validate(l.cfg.Distributor.ShardByAllLabels, l.cfg.Ingester.ActiveSeriesMetricsEnabled); err != nil {
-			return nil, err
+	targetStr := l.cfg.Target.String()
+	if strings.Contains(targetStr, All) || strings.Contains(targetStr, Distributor) || strings.Contains(targetStr, Querier) {
```

Comment:
@SungJin1212 Is querier here because it also use the shardByLabes to get user shard and query? Should we add ruler as well as it does the same?